Backend/prisma/schema.prisma

```diff
@@ -12,7 +12,11 @@ datasource db {
 
 model users {
```

Comment:
we need to rename the model here
```suggestion
model User {
```
All models should start with capital letter and in singular form,

Reply:
ok